src/vma/sock/sock-redirect.cpp

```diff
@@ -912,9 +917,9 @@ int close(int __fd)
 
 	srdr_logdbg_entry("fd=%d", __fd);
 
-	handle_close(__fd);
+	int close_now = handle_close(__fd);
```

Comment:
handle_close() return bool, but close_now is int. This may generate unnecessary warnings with some tools.

Reply:
done